Data Ingestion

```diff
@@ -0,0 +1,47 @@
+"""
```

Comment:
I'm not sure I understand why those scripts are needed. Instead, please add snippets directly into the corresponding folder, as python `.py` files and not at the root of the repo. Also, reference them in the `.md` as well.